<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegEntryDlg</class>
 <widget class="QDialog" name="RegEntryDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактирование объекта</string>
  </property>
  <property name="windowIcon">
   <iconset resource="reglisteditor.qrc">
    <normaloff>:/images/icons/Edit File-48.png</normaloff>:/images/icons/Edit File-48.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblIndex">
     <property name="text">
      <string>Индекс</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="sbIndex">
     <property name="prefix">
      <string>0x</string>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>8192</number>
     </property>
     <property name="displayIntegerBase">
      <number>16</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblType">
     <property name="text">
      <string>Тип</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cbType"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblName">
     <property name="text">
      <string>Имя</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="leName">
     <property name="maxLength">
      <number>254</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblDescr">
     <property name="text">
      <string>Описание</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPlainTextEdit" name="teDescr"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="bbOkCancel">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="reglisteditor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>bbOkCancel</sender>
   <signal>accepted()</signal>
   <receiver>RegEntryDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bbOkCancel</sender>
   <signal>rejected()</signal>
   <receiver>RegEntryDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
